build/signDSCModules.sh

```diff
@@ -64,7 +69,7 @@ DscModuleIntermediateDir="$IntermediateDir/merging_dsc_modules"
 DscModuleTargetDir="$TargetDir/dsc_signed"
 X64Dir="$BUILD_BASEDIR/Linux_ULINUX_1.0_x64_64_Release/dsc"
 X86Dir="$BUILD_BASEDIR/Linux_ULINUX_1.0_x86_32_Release/dsc"
-TestSigningDir=$1
+TestSigningDir=$2
```

Comment:
Updated the comments section to specify what is currently supported. Updated line#28 to be consistent with the rest of the script implementation (the way it validate the input parameter). Although existing style can be improved I am not going to re-write the script in this PR since it is only used by test tools. Definitely can come back to the script in the future.